cypress/integration/error-messages.test.ts

```diff
@@ -31,15 +31,15 @@ context("Test error messages", () => {
       errorBottom().should("contain.html", incompleteExpanded);
     });
 
-    it("renders incomplete for '+ a + b'", () => {
-      enterExpression("+ a + b");
-      computedNode().find(".error-short").should("contain.html", "Um, still working?");
-    });
-
-    it("renders incomplete for '+a'", () => {
-      enterExpression("+a");
-      computedNode().find(".error-short").should("contain.html", "Um, still working?");
-    });
+    const testIncomplete = (expression: string) => {
+      it (`renders incomplete for "${expression}"`, () => {
+        enterExpression(expression);
+        computedNode().find(".error-short").should("contain.html", "Um, still working?");
+      });
+    };
+    const incompleteStrings = ["+ a + b", "+a", "*a+b", "a+*b", "/a+b", "a+/b", "^a+b",
+      "a+^b", "(a+b", "a(+b", "a+(b", "a+b)"];
+    incompleteStrings.forEach(expression => testIncomplete(expression));
```

Comment:
I think the `it()` needs to be at the top level and the looping should happen within it.

Reply:
Thanks, that fixed the issue.